spec/suites/map/handler/Map.DragSpec.js

```diff
@@ -185,10 +203,8 @@ describe("Map.Drag", function () {
 			});
 			var mouse = hand.growFinger('mouse');
 
-			// We move 5 pixels first to overcome the 3-pixel threshold of
-			// L.Draggable.
-			mouse.wait(100).moveTo(200, 200, 0)
```

Comment:
Are you sure that removing `wait(100)` is good? I take the experience that it can sometimes results in errors if it is not set.

Reply:
I see no problem without wait, that's why I removed it.